core/src/main/scala/org/apache/spark/internal/config/ConfigBuilder.scala

```diff
@@ -147,6 +147,14 @@ private[spark] class TypedConfigBuilder[T](
     }
   }
 
+  /** Creates a [[ConfigEntry]] with a function has a default value */
```

Comment:
`Creates a [[ConfigEntry]] with a function has a default value`
->
`Creates a [[ConfigEntry]] with a function to determine the default value`